src/screens/VisualTests/SnapshotComparison.stories.tsx

```diff
@@ -86,12 +99,65 @@ export const FirstPassed: Story = {
   },
 } satisfies Story;
 
+export const StoryAdded: Story = {
+  args: {
+    tests: makeTests({
+      browsers: [Browser.Chrome, Browser.Safari],
+      viewports: [
+        {
+          status: TestStatus.Pending,
+          result: TestResult.Added,
+          viewport: 800,
+          comparisons: [
+            makeComparison({ result: ComparisonResult.Added, baseCapture: null }),
+            makeComparison({ result: ComparisonResult.Added, baseCapture: null }),
+          ],
+        },
+      ],
+    }),
+  },
+};
+
+export const ExistingStoryModeAdded: Story = {
+  args: {
+    tests: makeTests({
+      browsers: [Browser.Chrome, Browser.Safari],
+      viewports: [
+        {
+          status: TestStatus.Pending,
+          viewport: 800,
+          comparisons: [
+            makeComparison({ result: ComparisonResult.Added, baseCapture: null }),
+            makeComparison({ result: ComparisonResult.Equal }),
+          ],
+        },
+      ],
+    }),
+  },
+};
+
 export const ShowingBaseline: Story = {
   args: {
     baselineImageVisible: true,
   },
 } satisfies Story;
 
+export const NoBaseline: Story = {
```

Comment:
Maybe this should be called `StoryAddedInSelectedBuild` or something? I feel like there is a sequence of stories we should keep together:

- New story, not in any build
- New story, local build running (no latest build yet)
- New story, latest build on branch has it but not complete yet 
- New story, latest build on branch has it and has captured it
- New story, latest build has it and complete (selected build doesn't have it) [1]
- New story, selected build has it, but it isn't accepted yet.
- New story accepted

[1] Do we automatically switch to the latest build here? I don't know.

We don't necessarily need to resolve the exact behaviour in this PR but it'd be good to document the states.

Reply:
These stories might live in the `VisualTests` component, I was a little confused about what I was looking at.

Reply:
This test on this component was meant to capture that there was not toggle baseline button. The other tests live in Visual Tests as you noted. I added some of the new states in Visual Tests as best I could.

Reply:
> 🚧 New story, latest build on branch has it but not complete yet
> I'm not sure what this case would look like - would a test with status InProgress, can it have a result yet?

A test with `status: IN_PROGRESS` has no result.

The simple answer here would be the "new story UI" with an eyebrow about the latest build above, similar to [this one](https://www.chromatic.com/component?appId=6480e1b0042842f149cfd74c&csfId=screens-visualtests-visualtests--pending-local-build-capturing&buildNumber=992&k=650d5d272a1f6d351f9f835a-Right+2-up-up-false-false&h=10&b=-4), or [this one (when there is no local build)](https://www.chromatic.com/component?appId=6480e1b0042842f149cfd74c&csfId=screens-visualtests-visualtests--pending-ci-build-in-progress&buildNumber=992&k=650d5d272a1f6d351f9f835e-Right+2-up-up-false-false&h=11&b=-5)

> 🚧 New story, latest build has it and complete (selected build doesn't have it) [1]
> This isn't handled, but I added a story to capture it.

Sounds good.